docs/src/content/docs/commands/JSON.ARRLEN.md

```diff
@@ -1,137 +1,140 @@
 
-- `(error) ERR wrong number of arguments for 'JSON.ARRLEN' command`: This error occurs if the command is called with an incorrect number of arguments.
-- `(error) ERR key does not exist`: This error occurs if the specified key does not exist in the DiceDB database.
-- `(error) ERR path does not exist`: This error occurs if the specified path does not exist within the JSON document.
-- `(error) ERR path is not a JSON array`: This error occurs if the specified path does not point to a JSON array.
+4. **Path is not a JSON array**:
+   - Error Message: `(error) ERROR Path 'NON_ARRAY_PATH' does not exist or not array`
+   - Occurs when the specified path does not point to a JSON array.
 
-## Example Usage
+## Examples
 
 ### Example 1: Basic Usage
 
-Assume we have a JSON document stored at key `user:1001`:
+Setting a key `user:1001` with a JSON document:
 
-```json
-{
-  "name": "John Doe",
-  "emails": ["john.doe@example.com", "johndoe@gmail.com"],
-  "age": 30
-}
+```bash
+127.0.0.1:7379> JSON.SET user:1001 $ '{"name":"John Doe","emails":["john.doe@example.com","johndoe@gmail.com"],"age":30}'
+OK
 ```
 
 To get the length of the `emails` array:
 
-```plaintext
-JSON.ARRLEN user:1001 $.emails
-```
-
-`Response:`
-
-```plaintext
-2
+```bash
+127.0.0.1:7379> JSON.ARRLEN user:1001 $.emails
+(integer) 2
 ```
 
 ### Example 2: Root Path
 
-Assume we have a JSON document stored at key `user:1002`:
+Setting a key `user:1002` with a root array:
 
-```json
-[
-  "item1",
-  "item2",
-  "item3"
-]
+```bash
+127.0.0.1:7379> JSON.SET user:1002 $ '["item1", "item2", "item3"]'
+OK
 ```
 
 To get the length of the root array:
 
-```plaintext
-JSON.ARRLEN user:1002
+```bash
+127.0.0.1:7379> JSON.ARRLEN user:1002
+(integer) 3
 ```
 
-`Response:`
-
-```plaintext
-3
-```
+`Response: 3`
 
 ### Example 3: Non-Existent Path
 
-Assume we have a JSON document stored at key `user:1003`:
+Setting a key `user:1003` with a JSON document:
 
-```json
-{
-  "name": "Jane Doe",
-  "contacts": {
-    "phone": "123-456-7890"
-  }
-}
+```bash
+127.0.0.1:7379> JSON.SET user:1003 $ '{"name": "Jane Doe","contacts":{"phone":"123-456-7890"}}'
+OK
 ```
 
-To get the length of a non-existent array:
+To get the length of a non-existent `emails` array:
 
-```plaintext
-JSON.ARRLEN user:1003 $.emails
-```
-
-`Response:`
-
-```plaintext
-(null)
+```bash
+127.0.0.1:7379> JSON.ARRLEN user:1003 $.emails
+(error) ERROR Path '$.emails' does not exist
 ```
 
 ### Example 4: Path is Not an Array
 
-Assume we have a JSON document stored at key `user:1004`:
+Setting a key `user:1004` with a JSON document:
 
-```json
-{
-  "name": "Alice",
-  "age": 25
-}
+```bash
+127.0.0.1:7379> JSON.SET user:1004 $ '{"name": "Alice","age": 25}'
+OK
 ```
 
 To get the length of a non-array path:
 
-```plaintext
-JSON.ARRLEN user:1004 $.age
+```bash
+127.0.0.1:7379> JSON.ARRLEN user:1004 $.age
+(error) ERROR Path '$.age' does not exist or not array
 ```
 
-`Response:`
+### Example 5: Path with Wildcards
```

Comment:
We are not using the Example # prefix for the examples in our docs. Let's remove it for the sake of consistency.

Please do the same for the other occurrences too.
```suggestion
### Path with Wildcards
```

Reply:
Hi @JyotinderSingh, I thought numbering them would be better as we have many examples and they may even increase in future if we add support for complex JSONPaths. But, YES for the sake of consistency, I agree with this request. I will make this change.